src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -18673,11 +18673,33 @@ public static Vector512<T> Invoke(Vector512<T> x)
         /// <summary>T.PopCount(x)</summary>
         internal readonly struct PopCountOperator<T> : IUnaryOperator<T, T> where T : IBinaryInteger<T>
         {
-            public static bool Vectorizable => false; // TODO: Vectorize
+            public static bool Vectorizable => sizeof(T) == 1 && (AdvSimd.IsSupported || PackedSimd.IsSupported);
+
             public static T Invoke(T x) => T.PopCount(x);
-            public static Vector128<T> Invoke(Vector128<T> x) => throw new NotSupportedException();
-            public static Vector256<T> Invoke(Vector256<T> x) => throw new NotSupportedException();
-            public static Vector512<T> Invoke(Vector512<T> x) => throw new NotSupportedException();
+
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            public static Vector128<T> Invoke(Vector128<T> x)
+            {
+                Debug.Assert(sizeof(T) == 1);
+
+                if (AdvSimd.IsSupported)
+                {
+                    return AdvSimd.PopCount(x.AsByte()).As<byte, T>();
+                }
+
+                if (PackedSimd.IsSupported)
+                {
+                    return PackedSimd.PopCount(x.AsByte()).As<byte, T>();
+                }
+
+                throw new NotSupportedException();
```

Comment:
Maybe add a TODO for supporting #96162?

Reply:
Also, seems odd that it's throwing inside a method body that has been marked for inlining. Should this be extracted to a throw helper per usual practice?

Reply:
That branch shouldn't be reachable and should never end up in the resulting assembly code.  It's not worth using a throw helper, the purpose of which is to reduce the amount of IL/asm in the resulting code.  I can change it to an assert, though.

Reply:
> should never end up in the resulting assembly code

I think that eliminated branches still impact JITs decisions on inlining, not 100% sure about that though.